src/server/req_manager.c

```diff
@@ -1686,6 +1692,13 @@ mgr_queue_set(struct batch_request *preq)
 		rc = mgr_set_attr(pque->qu_attr, que_attr_def, QA_ATR_LAST,
 			plist, preq->rq_perm, &bad, (void *)pque,
 			ATR_ACTION_ALTER);
+		if (rc == PBSE_CANNOT_SET_ROUTE_QUE) {
```

Comment:
As suggested changed the error message to the following.

"Route queues are incompatible with the partition attribute"

Have updated EDD accordingly.